cosmwasm/contracts/crosschain-registry/src/state.rs

```diff
@@ -0,0 +1,22 @@
+use cosmwasm_std::Addr;
+use cw_storage_plus::{Item, Map};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+// CONTRACT_ALIAS_MAP is a map from a contract alias to a contract address
+pub const CONTRACT_ALIAS_MAP: Map<&str, String> = Map::new("contract_alias_map");
+
+// CHAIN_TO_CHAIN_CHANNEL_MAP is a map from a source<>destination chain pair to its respective channel id
+pub const CHAIN_TO_CHAIN_CHANNEL_MAP: Map<(&str, &str), String> =
+    Map::new("chain_to_chain_channel_map");
+
+// CHANNEL_TO_CHAIN_CHAIN_MAP is a map from a channel id on a source chain to its respective destination chain
+pub const CHANNEL_ON_CHAIN_CHAIN_MAP: Map<(&str, &str), String> =
+    Map::new("channel_to_chain_chain_map");
+
+pub const STATE: Item<State> = Item::new("state");
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct State {
```

Comment:
nit: I like calling these `Config`. Everything is state

Reply:
Agreed, changed! https://github.com/osmosis-labs/osmosis/pull/4320/commits/97be4979580e1b4dc2472366e3f4c34212461f0c